support-frontend/assets/components/legal/termsPrivacy/termsPrivacy.jsx

```diff
@@ -79,6 +80,19 @@ function TermsPrivacy(props: PropTypes) {
     );
   }
 
+  const isUSContributor = props.countryGroupId === 'UnitedStates';
+  const isNotOneOffContribution = props.contributionType !== 'ONE_OFF';
+  const referrerAcquisitionData = getReferrerAcquisitionData();
```

Comment:
We can get the `referrerAcquisitionData` from the redux store. The `getReferrerAcquisitionData` function should really only be called once, at the start.

So this component's props should instead receive a new field:
`referrerAcquisitionData: ReferrerAcquisitionData`

And the parent component can get this from redux in its `mapStateToProps` with:
`referrerAcquisitionData: state.common.referrerAcquisitionData`

Happy to pair on this!

Reply:
I figured this might be the case! I'll give you a shout if I get stuck, but hopefully can get this sorted tomorrow morning!

Reply:
This should be fixed now!